examples/merkle-tree-cli/src/commands.rs

```diff
@@ -0,0 +1,37 @@
+use clap::{Args, Parser, Subcommand};
+
+#[derive(Parser, Debug)]
+#[command(author = "Lambdaworks", version, about)]
+pub struct MerkleArgs {
+    #[clap(subcommand)]
+    pub entity: MerkleEntity,
+}
+
+#[derive(Subcommand, Debug)]
+pub enum MerkleEntity {
+    #[clap(about = "Generate a merkle tree")]
+    GenerateMerkleTree(GenerateMerkleTreeArgs),
```

Comment:
Just a personal opinion, maybe this could be just "GenerateTree", by context we already know that it is a merkle tree and the command becomes shorter :)

Reply:
Yes, the other two commands also ommit the word _Merkle_. Changing it.